src/ol/MapBrowserEventHandler.js

```diff
@@ -204,7 +204,7 @@ class MapBrowserEventHandler extends Target {
     // event.stopPropagation() or event.preventDefault().
     if (
       this.emulateClicks_ &&
-      !newEvent.propagationStopped &&
+      !newEvent.defaultPrevented &&
```

Comment:
Question: looking at the comment above, should we check `!(newEvent.defaultPrevented || newEvent.propagationStopped)` ?

Reply:
No - if we did, your issue would be back. I'm going to update the comment above.